Metrics.py

```diff
@@ -0,0 +1,52 @@
+class Metrics:
```

Comment:
I don't think y'all need to push this to master since we already have an updated copy.